src/Components/Header.js

```diff
@@ -38,33 +50,38 @@ export const StyledLinks = styled.a`
   line-height: 20px;
 `;
 
+/**
+ * Content Component
+ * @author [Pedro Nepomuceno](https://github.com/Pedro-Nepomuceno)
+ */
+
 function Header() {
   const { openModal } = useContext(ModalContext);
-  return (
-    <HeaderStyle
-      as={motion.header}
-      variants={sectionVariants}
-      initial="offscreen"
-      whileInView="onscreen"
-    >
-      <LogoImage src={headerLogo} alt="Sindano logo"></LogoImage>
-
-      <nav>
-        <StyledList>
-          <StyledIcons>
-            <StyledLinks href="#ourstory">About</StyledLinks>
-          </StyledIcons>
-          <StyledIcons>
-            <StyledLinks href="#whatwedo">What we do</StyledLinks>
-          </StyledIcons>
-          <StyledIcons>
-            <StyledLinks href="#faq">FAQ</StyledLinks>
-          </StyledIcons>
-        </StyledList>
-      </nav>
-
-      <PrimaryButton onClick={openModal} label="Request a call" />
-    </HeaderStyle>
-  );
+	return (
+		<>
+			<HeaderStyle
+				as={motion.header}
+				variants={sectionVariants}
+				initial="offscreen"
+				whileInView="onscreen">
+				<LogoImage src={headerLogo} alt="Sindano logo"></LogoImage>
+				<nav>
+					<StyledList>
+						<StyledIcons>
+							<StyledLinks href="#ourstory">About</StyledLinks>
+						</StyledIcons>
+						<StyledIcons>
+							<StyledLinks href="#whatwedo">What we do</StyledLinks>
+						</StyledIcons>
+						<StyledIcons>
+							<StyledLinks href="#faq">FAQ</StyledLinks>
+						</StyledIcons>
+					</StyledList>
+				</nav>
+				<Burger />
+				<PrimaryButton onClick={openModal} label="Request a call" />
```

Comment:
Hi @Pedro-Nepomuceno ! This code is updated, so make sure to `git pull` from this branch before making adjustments. 

The `Primary Button` still shows up on the smallest screen size along with the `Burger` menu.